jdtls.ext/com.microsoft.jdtls.ext.core/src/com/microsoft/jdtls/ext/core/PackageCommand.java

```diff
@@ -106,14 +106,13 @@
     }
 
     /**
-     * Resolve the path for Java file URI
+     * Resolve the path for Java file URI.
      *
      * @param arguments
      *            List of the arguments which contain one entry of the target
      *            compilation unit URI.
-     *
      * @return the list of the path
-     * @throws CoreException
+     * @throws CoreException when create node or get resource
```

Comment:
`when fail to ...` or `if an error occurred when...`
Please also check other modified `@throws` to avoid any misunderstanding.